src/rules/commentFormatRule.ts

```diff
@@ -119,4 +127,21 @@ class CommentWalker extends Lint.RuleWalker {
         }
     }
 
+    private startsWithUppercase(commentText: string): boolean {
```

Comment:
This shares a great deal of logic with `startsWithLowercase`. The two methods should be refactored for DRY.